### PR TITLE
Remove bold tags from search results

### PR DESCRIPTION
**Pull request by cptran777**
The tags were caused by legacy code that was creating bold tags "< b >" around the keyword being searched in the results. However, in our current ember implementation this tag does not get rendered as html but is instead a visible text string. Refactor function to remove unnecessary bold tags and keep its schema shortening functionality. Global shouldn't be necessary anymore.

`ember test` results:
```
ok 1 Chrome 66.0 - ESLint | mirage: mirage/data/schema.js
ok 2 Chrome 66.0 - ESLint | mirage: mirage/factories/access.js
...
ok 368 Chrome 66.0 - Unit | Utility | validators/urn: buildLiUrn
ok 369 Chrome 66.0 - ember-qunit: Ember.onerror validation: Ember.onerror is functioning properly

1..369
# tests 369
# pass  363
# skip  6
# fail  0

# ok
```

### CODE REVIEW

File: wherehows-web/vendor/legacy_sources/global_functions.js [hunk 1]
```diff
@@ -375,29 +375,6 @@ function updateActiveTab() {
 var scrollToTreeNode = function () {
   $("#tabSplitter").scrollTo($('.fancytree-focused'), 800)
 }
-function highlightResults(result, index, keyword) {
```
Comment (theseyi) [inline wherehows-web/vendor/legacy_sources/global_functions.js]: 👍

File: wherehows-web/app/utils/api/datasets/highlight-results.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import { IDataset } from 'wherehows-web/typings/api/datasets/dataset';
+
```
Comment (theseyi) [inline wherehows-web/app/utils/api/datasets/highlight-results.ts]: Maybe i'm missing it, but does this actually highlight the keywords?

Comment (cptran777) [reply wherehows-web/app/utils/api/datasets/highlight-results.ts]: It used to, but I guess the naming is kind of misleading now.

File: wherehows-web/app/routes/search.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { isBlank } from '@ember/utils';
 import $ from 'jquery';
 import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
 import buildUrl from 'wherehows-web/utils/build-url';
-import highlightResults from 'wherehows-web/utils/api/datasets/highlight-results';
+import createSearchEntries from 'wherehows-web/utils/datasets/create-search-entries';
```
Comment (theseyi) [inline wherehows-web/app/routes/search.js]: @cptran777 I'm still not sure what the intent of this function is, the original implementation was to highlight the keywords in the search result entries with bold tags, this doesn't seem to do anything of that sort? Is that the case?

**Pull request merged**